Cargo.toml

```diff
@@ -51,7 +51,7 @@ dora-coordinator = { version = "0.2.2", path = "binaries/coordinator" }
 
 [package]
 name = "dora-examples"
-version = "0.0.0"
+version = "0.1.0"
```

Comment:
We're not planning on releasing this. Please remove this change.

Reply:
Rolled back to previous version number.